packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -1,35 +1,39 @@
 <% if (isModelBaseBuiltin) { -%>
-import {<%= modelBaseClass %>, model, property} from '@loopback/repository';
-<% } else { -%>
-import {model, property} from '@loopback/repository';
-import {<%= modelBaseClass %>} from '.';
-<% } -%>
+  import {<%= modelBaseClass %>, model, property} from '@loopback/repository';
```

Comment:
Please revert all whitespace-only changes in this file.

Reply:
I didn't change this lines but I think that this is the not isModelBaseBuiltin case and modelBaseClass import should be from current folder, not from loopback/repository

Reply:
Sorry, I didn't understand. Reverted.

Reply:
Ditto, I don't see these changes being reverted.